core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableInfo.java

```diff
@@ -68,17 +70,10 @@
    */
   private long lastUpdatedTime;
 
-  /**
-   * metadata file path (check if it is really required )
-   */
-  private String metaDataFilepath;
-
-  /**
-   * store location
-   */
+  @Obsolete
```

Comment:
Why it is obsolete ? is there any other way user can get tablePath?

Reply:
fixed, it is not obsoleted